src/backend/pluginLoaders.js

```diff
@@ -31,12 +32,24 @@ opaque type CachedProject = {|
   +project: Project,
 |};
 
+/**
+ * Represents all disjoint WeightedGraphs for a CachedProject.
```

Comment:
"all disjoint" -- are we guaranteeing that PluginGraphs need to be disjoint now?

Reply:
Well, specifically we require that `WeightedGraph.merge(graphs)` doesn't crash it, or it would be a bug. So not it's not a type guarantee, more an assumption.

Reply:
Sure, but `WeightedGraph.merge` works on non-disjoint graphs. As a silly example, given that `g` is the WeightedGraph for a GitHub project, `WeightedGraph.merge([g,g])` returns a copy of `g`.

So let's fix the comment. Just saying it represents the plugin-generated WeightedGraphs for a CachedProject is likely fine.

Reply:
Wouldn't the Github WeightedGraph have the plugin's `weightsForDeclaration`? Meaning, same address weights, meaning it would throw without a resolver?

Reply:
Oh, you're right, it would.